k2/csrc/fsa_equivalent.h

```diff
@@ -15,10 +16,11 @@
 namespace k2 {
 
 /*
-  Returns true if the Fsa `a` is equivalent to `b`.
-  CAUTION: this one will be quite hard to implement.
+  Returns true if the Fsa `a` is stochastically equivalent to `b` by randomly
+  generating `npath` paths from one of them and then checking if the
+  paths exist in the other one.
  */
-bool IsEquivalent(const Fsa &a, const Fsa &b);
+bool IsRandEquivalent(const Fsa &a, const Fsa &b, std::size_t npath = 100);
```

Comment:
Why is `npath` of type `std::size_t`?

Reply:
as it is the number of paths we will generate, a positive value